Lib/test/test_logging.py

```diff
@@ -1591,6 +1591,30 @@ def test_logger_disabling(self):
         self.apply_config(self.disable_test, disable_existing_loggers=False)
         self.assertFalse(logger.disabled)
 
+    def test_config_set_handler_names(self):
+        test_config = """
+            [loggers]
+            keys=root
+
+            [handlers]
+            keys=hand1
+
+            [formatters]
+            keys=form1
+
+            [logger_root]
+            handlers=hand1
+
+            [handler_hand1]
+            class=StreamHandler
+            formatter=form1
+
+            [formatter_form1]
+            format=%(levelname)s ++ %(message)s
+            """
+        self.apply_config(test_config)
+        assert logging.getLogger().handlers[0].name == 'hand1'
```

Comment:
Please use self.assertEquals

Reply:
Actually, `assertEqual` (no `s` at the end)